docs/content/direct/pr-signoff.md

```diff
@@ -1,8 +1,64 @@
-# How to Sign-off on Your Pull Requests
+# Git Commit Signoff and Signing
 
-In order to get your pull requests approved, you must first complete a DCO sign-off. This process is defined by the CNCF, and there are two cases: individual contributors and contributors that work for a corporate CNCF member. To do this as an individual contributor, you must have a GPG and SSH key. Basic setup instructions can be found below (For more detailed instructions, refer to the Github [GPG](https://docs.github.com/en/authentication/managing-commit-signature-verification/generating-a-new-gpg-key) and [SSH](https://docs.github.com/en/authentication/connecting-to-github-with-ssh/generating-a-new-ssh-key-and-adding-it-to-the-ssh-agent#generating-a-new-ssh-key) setup pages):
+**NOTE**: "sign-off" is different from "signing" a commit or tag.  The
+former indicates your assent to the repository's terms for
+contributors, the latter adds a cryptographic checksum that is rarely
+displayed.  See [the git
+book](https://git-scm.com/book/en/v2/Git-Tools-Signing-Your-Work)
+about signing. For commit signoff, do a web search on `git
+signoff`. GitHub has a concept of [a commit being
+"verified"](https://docs.github.com/en/authentication/managing-commit-signature-verification)
+that extends the Git concept of signing.
 
----
+In order to get your pull requests approved, you must first complete a
+DCO sign-off. This process is defined by the CNCF, and there are two
+cases: individual contributors and contributors that work for a
+corporate CNCF member. Both mean consent with the terms stated in the
+`DCO` file at the root of this Git repository.
+
+In the case of an individual, DCO sign-off is done with a Git "signoff".
+
+We prefer that commits contributed to this repository be signed and
+GitHub verified, but this is not strictly necessary or enforced.
+
+## Commit Sign-off
+
+Your submitted PR must pass the automated checks in order to be reviewed. One of these checks that each commit that you propose to contribute is signed-off. If you use the `git` shell command, this involves passing the `-s` flag on the command line. For other tools, consult their documentation.
+
```

Comment:
add a quick code block showing how '-s' is incorporated in the full command line

Reply:
done